src/main/java/com/parasoft/findings/sonar/sensor/CoverageSensor.java

```diff
@@ -109,10 +111,12 @@ public void uploadFileCoverageData(File report, SensorContext context) {
             Document document = reader.read(report);
 
             Element root = document.getRootElement();
-            Element packagesElement = root.element("packages");
-
-            for (Element packageElement : packagesElement.elements("package")) {
-                for (Element classElement : packageElement.element("classes").elements("class")) {
+            List<Element> packageElements = root.element("packages").elements("package");
```

Comment:
We need to check null situation for ```root.element("packages")```, users may pass a .xml
file but not a coverage report file.